tailwind.config.js

```diff
@@ -15,6 +15,56 @@ module.exports = {
           800: '#08072e',
         },
       },
+      typography: ({ theme }) => ({
+        DEFAULT: {
+          css: {
+            '--tw-prose-body': theme('colors.brand.100'),
+            '--tw-prose-headings': theme('colors.brand.100'),
+            '--tw-prose-bold': theme('colors.brand.100'),
+            '--tw-prose-links': theme('colors.brand.300'),
+            '--tw-prose-quotes': theme('colors.brand.100'),
+            '--tw-prose-quote-borders': theme('colors.brand.300'),
+            '--tw-prose-counters': theme('colors.brand.100'),
+            '--tw-prose-bullets': theme('colors.brand.100'),
+            '--tw-prose-code': theme('colors.brand.100'),
+            '--tw-prose-pre-bg': theme('colors.brand.700'),
+            '--tw-prose-pre-code': theme('colors.brand.300'),
+
+            a: {
+              textDecoration: 'none',
+              '&:hover': {
+                textDecoration: 'underline',
+              },
+            },
+
+            code: {
+              '&::before, &::after': {
+                content: 'none !important',
+              },
+            },
+
+            pre: {
+              borderWidth: '1px',
+              borderStyle: 'solid',
+              borderColor: theme('colors.brand.600'),
+            },
+
+            iframe: {
+              width: '100%',
+
+              '&::focus-visible': {
+                outline: 'none',
+              },
```

Comment:
What does this do? (And if this is necessary, remember that by setting `ouline: 'none'` we probably just messed with a11y.